File: src/main/java/no/nav/service/pensjon/regler/to/HentGyldigSatsRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.Date;


import no.nav.domain.pensjon.regler.kode.SatsTypeCti;

public class HentGyldigSatsRequest extends ServiceRequest {
	private Date fomDato;
	private Date tomDato;
	private SatsTypeCti satsType;
    private boolean beregnetSomGift;
    private boolean forsorgerEPSOver60;
    private boolean ungUfor;

}

File: src/main/kotlin/no/nav/pensjon/regler/to/HentGyldigSatsRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.enum.SatsTypeEnum
import java.util.*
import java.time.LocalDate

class HentGyldigSatsRequest : ServiceRequest() {
    var fomDatoLd: LocalDate? = null
    var tomDatoLd: LocalDate? = null
    var satsTypeEnum: SatsTypeEnum? = null
    var beregnetSomGift = false
    var forsorgerEPSOver60 = false
    var ungUfor = false
}